packages/RobotRaconteur/meta.yaml

```
@@ -5,6 +5,8 @@ package:
 source:
   url: https://github.com/robotraconteur/robotraconteur/releases/download/v1.2.2/RobotRaconteur-1.2.2-Source.tar.gz
   sha256: "837710102b5ee6ad81a22e14b313b1dd09995c3da3e7846355866553b3c52715"
+  patches:
+    - pyproject_toml.patch
```

If you're adding an entire file you can do:
```suggestion
  extras:
    - [pyproject.toml, pyproject.toml]
```
And then just add the file itself as `packages/RobotRaconteur/pyproject.toml`.

---

Thanks I made the suggested change to use "extras" section